src/app/shared/utils/utils.ts

```diff
@@ -324,9 +324,7 @@ export async function asyncEvery(arr: any[], predicate: Function) {
   return true;
 }
 
-/** A recursive version of Partial, making all properties, included nested ones, optional.
```

Comment:
nit(non-blocking)
I'm also wondering why we no longer need the RecursivePartial type - seemed like a nice utility to have. Has it been moved or just no longer required?